### PR TITLE
Add UUID to Product ID of Manifests

### PR DESCRIPTION
**Pull request by rjohnson2011**
## Description
Modifies the `productId` value of the `manifest.json` files of various products to reflect the UUID detailed in the [product directory](https://docs.google.com/spreadsheets/d/1VaXAZIYj2LdG4H9ZQUbdCGjMk5qQ8Ari2mMySF1oah0/edit#gid=0).

Note: Before merging please advise on appropriate ProductID for the following products which contain lines to the same path:
```Secure Inbox and Resource & Support
Original Claims and Benefits Delivery at Discharge
```

## Original issue(s)
[Zenhub - Update Manifests to add UUID#41040](https://app.zenhub.com/workspaces/sprint-board-qa-standards-613a3f47e06ba00013d05eed/issues/department-of-veterans-affairs/va.gov-team/41040)

## Testing done
Ticket is in review for QA Standards and Release Tools to examine.

## Acceptance criteria
- [x] All productIds in vets-website are using the UUIDs defined in the product list

### CODE REVIEW

**Review by ddzz — APPROVED** [review]

**Comment by rjohnson2011** [comment]
Closing temporarily until the codeowners file is merged frist.

**Pull request closed without merging**
